### DISCUSSION TITLE
Implement real multi tenant

### DISCUSSION THREAD

**Discussion by ks-updev** [blitz-js/blitz#3328] [Q&A]
Just want to know is there a way to create a separate database architecture in the blitzjs?
I don't like the approach on https://blitzjs.com/docs/multitenancy, obviously it is simple but it likes a walkaround for me.
References: https://web.archive.org/web/20191125145923/http://cloudcomputing.sys-con.com:80/node/1610582

**Comment by noxify** [comment]
Maybe this could help: https://github.com/Errorname/prisma-multi-tenant

Source: https://github.com/prisma/prisma/discussions/2846
